src/utils.ts

```diff
@@ -47,4 +47,21 @@ export function tryNumber(n: string | undefined): number | undefined {
   }
 }
 
-export const readFile = util.promisify(fs.readFile);
+export function resolvePath(path: string, home: string = os.homedir()): string {
+  if (home)
+    return path.replace(/^~(?=$|\/|\\)/, home)
+  return path;
+}
+
+export async function readFile(path: string, options: object | string): Promise<string> {
+  return new Promise(
+    (okay, fail) => {
+      fs.readFile(resolvePath(path), options, (err, data) => {
+        if (err)
+          return fail(err);
+        if (data instanceof Buffer)
```

Comment:
The `'utf8'` second param, I believe, returns the result as a string instead of a buffer.